packages/genesys-spark-components/src/components/beta/gux-side-panel/components/gux-side-panel-heading/gux-side-panel-heading.tsx

```diff
@@ -0,0 +1,47 @@
+import { Component, Element, h, Prop } from '@stencil/core';
+import { trackComponent } from '@utils/tracking/usage';
+import {
+  GuxSidePanelHeadingLevel,
+  GuxSidePanelHeadingTag
+} from '../../gux-side-panel.types';
+
+@Component({
+  tag: 'gux-side-panel-heading',
+  styleUrl: 'gux-side-panel-heading.scss',
+  shadow: true
+})
+export class GuxSidePanelHeading {
+  @Element()
+  private root: HTMLElement;
+  /**
+   * Heading level, 1-6.
+   */
+  @Prop()
+  level: GuxSidePanelHeadingLevel = 1;
+
+  @Prop()
+  text: string;
```

Comment:
I am thinking there maybe drawbacks to this potentially or is it still being used ?